pypuf/simulation.py

```diff
@@ -113,29 +116,30 @@ class NoisyLTFArray(LTFArray):
     Class that simulates k LTFs with n bits and a constant term each 
     with noise effect and constant bias (=scale) added.
     """
-    def __init__(self, weight_array, transform, combiner, scale, noise):
+    def __init__(self, weight_array, transform, combiner, bias, scale, intensity=0.1):
         """
         Initializes LTF array with bias and noise effect based on inputs.
         The noise effect equals a normal distributed random variable
-        with mu=0, sigma= scale of overall weights * noise.
+        with mu=0, sigma= scale of overall weights * intensity.
         The bias is a constant value out of a normal distribution.
```

Comment:
The bias should be an (k × 1)-array. Just like for the `weight_array`, the user may want to choose the bias for himself and not rely on random bias.